modules/standard/IO.chpl

```diff
@@ -4557,7 +4556,7 @@ proc channel.read(type t ...?numTypes) where numTypes > 1 {
 
 // documented in style= error= version
 pragma "no doc"
-inline proc channel.write(args ...?k, out error:syserr):bool {
+inline proc channel.write(const args ...?k, out error:syserr):bool {
```

Comment:
Right now, we're not tracking const-ness within tuple elements. So you can possibly do things like capture a ref to a const array in a tuple formal argument and then modify it.
I think it's reasonable for documentation to use the `const` intent on these `write` functions but it shouldn't need to be there, long-term.